src/sentry/replays/models.py

```diff
@@ -16,7 +16,8 @@ class ReplayRecordingSegment(Model):
     project_id = BoundedBigIntegerField()
     replay_id = models.CharField(max_length=32, db_index=True)
     file_id = BoundedBigIntegerField(db_index=True)
-    # recordings have multiple segments, ordered by their sequence_id
+    # recordings have multiple segments, ordered by their segment_id
+    # TODO: rename sequence_id to segment_id
     sequence_id = BoundedIntegerField()
```

Comment:
Column needs a name change and migration.  Looks like this pull was closed: https://github.com/getsentry/sentry/pull/37090/files#diff-87b3cfe1dbf66368a35666aab7253de1ae7240288e29ebd70afea9c444bd40fe

Reply:
hi colton, see PR description:
```
i'd like to rename the django model name (while keeping the postgres column the same as outlined [here](https://develop.sentry.dev/database-migrations/#renaming-columns), but i couldn't get the migrations to generate correctly, so added a TODO for now, and can swap that later.
```

Reply:
Are you trying to rename the column in code or in the database?  The `db_column` attribute should not generate any migrations.

Reply:
in theory yes, it shouldn't, but it is. need to dig into that, was planning on doing it after our initial ship.